source/manual/alerts/chat-ai-alerts.html.md

```diff
@@ -31,16 +31,19 @@ The 503 error code has been excluded from being alerted on, as that will be used
 - HighPodCPUWorker
 - HighPodMemoryFE
 - HighPodMemoryWorker
-- SidekiqQueueLength
-- SidekiqJobAge
+- SidekiqAnswerQueueLength
+- SidekiqAnswerJobAge
+- SidekiqDefaultJobAge
 - govuk-chat-bedrock-token-threshold-50
 - govuk-chat-bedrock-token-threshold-100
 
 The `LongRequestDuration` alert indicates that a backend part of the service is not responding as expected, and as a result the user experience is severely impacted. For example, we have seen it fire when the RDS Postgres Database is offline.
 
 Any of the `High*` alerts relate to EKS Pod performance. Horizontal Pod Autoscaling has been employed, so along with the rate limiting in place, we should not see these trigger but if they do, it could indicate an issue with the EKS cluster. HPA configuration can be found in the relevant `values-{environment}.yaml` files [here].
 
-`SidekiqQueueLength` is used as the metric to scale the worker pods, so an increase in this will likely be down to high load. If the age of the oldest job in the Sidekiq queue is increasing, it may be a process has got stuck.
+`SidekiqAnswerQueueLength` is used as the metric to scale the worker pods, so an increase in this will likely be down to high load. If the age of the oldest job in the Sidekiq answer queue is increasing, it may be a process has got stuck.
+
+The `SidekiqAnswerJobAge` and `SidekiqDefaultJobAge` alerts indicate that a Sidekiq job has been in the queue for for over 15 minutes or 6 hours respectively.
```

Comment:
I think we might want to expand here to explain a bit why we've got such different values here. For SidekiqAnswerJobAge we aim to not have items waiting in a queue and expect infra to scale to try avoid it, so building up a queue here is a sign of diminished user experience.

Whereas for 6 hours, we're really just wanting some indicator that it looks like an concerningly large backlog has built up that should be investigated.